Sources/BottomSheet/Helper/AppleScrollBehaviourScrollView/UIScrollViewWrapper.swift

```diff
@@ -160,15 +160,23 @@ internal struct UIScrollViewWrapper<Content: View>: UIViewControllerRepresentabl
             0
         )
         let displacementY = viewController.scrollView.contentOffset.y - restOffsetY
-        let threshold = 0.5 / UIScreen.main.scale
+        
+#if os(visionOS)
+        let scale = viewController.traitCollection.displayScale
+#else
+        let scale = UIScreen.main.scale
+#endif
+        let threshold = 0.5 / scale
         
         let duration: TimeInterval = {
             if abs(displacementY) == 0 && abs(velocityY) == 0 {
                 return 0
             }
             
             let timeInterval1 = 1 / 10 * log(2 * abs(displacementY) / threshold)
-            let timeInterval2 = 2 / 10 * log(4 * abs(velocityY + 10 * displacementY) / (CGFloat(M_E) * 10 * threshold))
+            
+            let interval2Threshold = (CGFloat(M_E) * 10 * threshold)
```

Comment:
the compiler failed so I needed to separate the complexity. Different name is probably preferred but I figure I'd leave that up to you

Reply:
LGTM